lucille-plugins/lucille-api/README.md

```diff
@@ -0,0 +1,51 @@
+# Lucille API
+
+## Running the API
+
+1. Navigate into the `lucille-admin` module
+2. Run `mvn clean install` to generate the admin jar
+
+The API can be run locally or in a Docker container:
+
+### Local Run:
+1. Replace the placeholders with your config file names or export them as environment variables. See [Configuration](#Configuration) for more information
+2. `java -Dconfig.file=${LUCILLE_CONF} -jar target/lucille-api-0.2.3-SNAPSHOT.jar server ${DROPWIZARD_CONF}`
```

Comment:
should probably replace the 0.2.3 in the jar name with a placeholder like "<version>" because this is going to go out of date and we don't want to have to update it with each release